<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Demo</class>
 <widget class="QMainWindow" name="Demo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Onglet Texte</string>
       </attribute>
       <widget class="QLabel" name="texte_histoire">
        <property name="geometry">
         <rect>
          <x>391</x>
          <y>9</y>
          <width>361</width>
          <height>501</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>376</width>
          <height>497</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QSpinBox" name="compteur_amis">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>90</y>
           <width>171</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
        </widget>
        <widget class="QLineEdit" name="ligne_heros">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>30</y>
           <width>171</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>(héros)</string>
         </property>
        </widget>
        <widget class="QDial" name="cadrant_temperature">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>230</y>
           <width>131</width>
           <height>151</height>
          </rect>
         </property>
         <property name="minimum">
          <number>-20</number>
         </property>
         <property name="maximum">
          <number>40</number>
         </property>
        </widget>
        <widget class="QLabel" name="texte_histoire_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>131</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Nom du héros</string>
         </property>
        </widget>
        <widget class="QLabel" name="texte_histoire_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>90</y>
           <width>131</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Nom d'amis</string>
         </property>
        </widget>
        <widget class="QDateEdit" name="date_depart">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>150</y>
           <width>171</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="minimumDate">
          <date>
           <year>2024</year>
           <month>1</month>
           <day>1</day>
          </date>
         </property>
         <property name="date">
          <date>
           <year>2024</year>
           <month>2</month>
           <day>17</day>
          </date>
         </property>
        </widget>
        <widget class="QTimeEdit" name="heure_gouter">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>200</y>
           <width>171</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="maximumTime">
          <time>
           <hour>22</hour>
           <minute>0</minute>
           <second>0</second>
          </time>
         </property>
         <property name="minimumTime">
          <time>
           <hour>18</hour>
           <minute>0</minute>
           <second>0</second>
          </time>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
         <property name="time">
          <time>
           <hour>18</hour>
           <minute>0</minute>
           <second>0</second>
          </time>
         </property>
        </widget>
        <widget class="QLabel" name="texte_histoire_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Date de Départ</string>
         </property>
        </widget>
        <widget class="QLabel" name="texte_histoire_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>200</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Heure du goûter</string>
         </property>
        </widget>
        <widget class="QLabel" name="texte_temperature">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>280</y>
           <width>171</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Température (0)</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Onglet Couleur</string>
       </attribute>
       <widget class="QSlider" name="glisseur_rouge">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>180</y>
          <width>160</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>50</number>
        </property>
       </widget>
       <widget class="QSlider" name="glisseur_vert">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>250</y>
          <width>160</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>50</number>
        </property>
       </widget>
       <widget class="QSlider" name="glisseur_bleu">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>320</y>
          <width>160</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>50</number>
        </property>
       </widget>
       <widget class="QLabel" name="texte_couleur">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>60</y>
          <width>391</width>
          <height>361</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="texte_rouge">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>140</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Rouge (0)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="texte_vert">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>210</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Vert (0)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="texte_bleu">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>290</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Bleu (0)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Onglet Final</string>
       </attribute>
       <widget class="QPushButton" name="bouton_papillon">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>30</y>
          <width>551</width>
          <height>371</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Papillon 1.jpg</normaloff>Papillon 1.jpg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>480</width>
          <height>480</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="texte_papillon">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>410</y>
          <width>501</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Papillon 1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="url_papillon">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>450</y>
          <width>771</width>
          <height>41</height>
         </rect>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
